ui/src/app/auth/auth-roles.ts

```diff
@@ -0,0 +1,5 @@
+// Default values of OSMT Roles
+export const OSMT_ADMIN = "ROLE_Osmt_Admin"
+export const OSMT_CURATOR = "ROLE_Osmt_Curator"
```

Comment:
Isn't there a ROLE_Osmt_View?

Reply:
Good point.  Roberto's implementation seems to assume that's the default role.